mne/channels/montage.py

```diff
@@ -73,9 +73,9 @@ def __repr__(self):
         return s
 
     @copy_function_doc_to_method_doc(plot_montage)
-    def plot(self, scale_factor=20, show_names=False, show=True):
-        return plot_montage(self, scale_factor=scale_factor,
-                            show_names=show_names, show=True)
+    def plot(self, kind='3d', scale_factor=20, show_names=False, show=True):
```

Comment:
you cannot change the API like that. If you add a parameter to a public function add it to the end.